nautobot/extras/tests/integration/test_tagfilter.py

```diff
@@ -31,8 +32,9 @@ def tearDown(self):
     def test_tag_matching_content_type(self):
         # Navigate to the Provider list view
         self.browser.links.find_by_partial_text("Circuits").click()
-        self.browser.links.find_by_partial_text("Providers").click()
-
+        link = self.browser.links.find_by_partial_text("Providers").first
+        self.browser.execute_script("arguments[0].scrollIntoView(true);", link._element)
+        link.click()
```

Comment:
This was amended with help from Daria, because selenium was not entering Providers page and staying on the Home Page.